### ISSUE TITLE
Flutter 3.0 changes

### ISSUE THREAD

**Issue by zawadZwd** [lulupointu/vrouter#193]
Hi

After new release of Flutter, the plugin has not working at all because of this error:

```dart

../../Flutter/.pub-cache/hosted/pub.dartlang.org/vrouter-1.2.0+21/lib/src/widgets/anchor/vanchor.dart:123:22: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
                     ^
```

**Comment by lulupointu** [comment]
I'm looking into this

**Comment by lulupointu** [comment]
Fixed in `1.2.1`.

Can you confirm ?

**Comment by zawadZwd** [comment]
I do. Error disappeared 😎
